x/ibc-rate-limit/contracts/rate-limiter/src/msg.rs

```diff
@@ -0,0 +1,73 @@
+use cosmwasm_std::Addr;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct Channel {
+    pub name: String,
+    pub quotas: Vec<QuotaMsg>,
+}
```

Comment:
```suggestion
// The channel struct contains a name representing a unique identifier within ibc-go, and a list of rate limit quotas
#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
pub struct Channel {
    pub name: String,
    pub quotas: Vec<QuotaMsg>,
}
```

Reply:
A bit unclear to me how this is used from the msg file, still working through the entire contract